etc/default/who.ini

```diff
@@ -1 +0,0 @@
-/usr/lib/ckan/default/src/ckan/who.ini
```

Comment:
this is no longer required as it is handled in docker during build with a command